db/schema.rb

```diff
@@ -433,7 +433,6 @@
     t.integer  "parent_id",       limit: 4
     t.string   "description",     limit: 255,   default: "", null: false
     t.integer  "default_page_id", limit: 4
-    t.text     "cache",           limit: 65535
```

Comment:
I think I know the reason why the travis ci failed. Do you manually delete L436? If so, it is not correct. You have to commit the `schema.rb` after running migrations.